dpl-docs/views/layout.dt

```diff
@@ -32,97 +33,182 @@ html(lang='en-US')
-              | local clone.
+            .tip.smallprint
+              - auto pagename = modname ? path_prefix ~ filename : "index" ;
+                a(href="https://issues.dlang.org/enter_bug.cgi?bug_file_loc=http%3A%2F%2Fdlang.org/library/#{pagename}.html&bug_severity=enhancement&component=#{project}&op_sys=All&priority=P3&product=D&rep_platform=All&short_desc=%5B#{title}%5D&version=D2") Report a bug
+              div
+                | If you spot a problem with this page, click here to create a
+                | Bugzilla issue.
 
-          //- a.tip.button(href='http://www.prowiki.org/wiki4d/wiki.cgi?DocComments/Phobos/#{info.node.qualifiedName}')
-          a.tip.button(href='http://wiki.dlang.org/DocComments/Phobos/#{info.node.qualifiedName}')
-            | Page wiki
-            span
-              | View or edit the community-maintained wiki page associated with this page.
+            - if( modname )
+              .tip.smallprint
+                a(href='https://github.com/D-Programming-Language/#{project}/edit/master/#{path_prefix}#{filename}.d')
+                  | Improve this page
+                div
+                  | Quickly fork, edit online, and submit a pull request for this page.
+                  | Requires a signed-in GitHub account. This works well for small changes.
+                  | If you'd like to make larger changes you may want to consider using
+                  | local clone.
 
-      - auto hidx = std.string.lastIndexOf(title, ' ');
-      - auto node = info.docGroups.length ? info.docGroups[0].members[0] : null;
-      - if (title.endsWith(" - multiple declarations") && node)
-        h1 <code><a href="#{info.linkTo(info.mod)}">#{info.mod.qualifiedName}</a>.#{title[0 .. $-24]}</code> #{title[$-24 ..$]}
-      - else if (hidx > 0 && !title.startsWith("Module ") && node)
-        h1 #{title[0 .. hidx]} <code><a href="#{info.linkTo(node.parent)}">#{node.parent.qualifiedName}</a>.#{node.name}</code>
-      - else if (hidx > 0 && title.startsWith("Module ") && info.mod)
-        h1 Module <code>#{info.mod.qualifiedName}</code>
-      - else
-        h1= title
-      block body
+        - auto hidx = std.string.lastIndexOf(title, ' ');
+        - auto node = info.docGroups.length ? info.docGroups[0].members[0] : null;
+        - if (title.endsWith(" - multiple declarations") && node)
+          h1 <code><a href="#{info.linkTo(info.mod)}">#{info.mod.qualifiedName}</a>.#{title[0 .. $-24]}</code> #{title[$-24 ..$]}
+        - else if (hidx > 0 && !title.startsWith("Module ") && node)
+          h1 #{title[0 .. hidx]} <code><a href="#{info.linkTo(node.parent)}">#{node.parent.qualifiedName}</a>.#{node.name}</code>
+        - else if (hidx > 0 && title.startsWith("Module ") && info.mod)
+          h1 Module <code>#{info.mod.qualifiedName}</code>
+        - else
+          h1= title
+        block body
 
-      |     
-      #quickindex.quickindex
+        |     
+        #quickindex.quickindex
```

Comment:
> We use static file generation on dlang.org though.

Right. What I have in mind might work then: Generate a .ddoc file and run dmd on it from the .dt template. Otherwise I don't know how to get the correct root dir set up.

I'm going to look further into this tomorrow.